druntime/src/object.d

```diff
@@ -104,6 +104,14 @@ else version (AArch64)
     else version = WithArgTypes;
 }
 
+// Lower Associative Array to a newaa struct for static initialization.
```

Comment:
Doc comment please.

Reply:
This isn't really meant to be a public function.

In fact, I'm actually thinking, we should rename this with a `_d_` in front to let people know this is a compiler hook and not for general use.